packages/api/src/validation/ajvAPIValidator.ts

```diff
@@ -23,7 +23,8 @@ export class AjvAPIValidator<T extends AjvSchema> implements APIValidator {
     const ajv = new Ajv({
       allowUnionTypes: true,
       useDefaults: config.mutateWithDefaultValues,
-      coerceTypes: true,
+      // coerce numbers, boolean values AND array
```

Comment:
It's great that this feature exists! Despite the comment, I still had to look up the documentation to understand what this does. Maybe: "Coerce incoming values to the schema-specified types. When the schema specifies an array type, will map `x` to `[x]` as needed."

Reply:
I like that. I will update!